### DISCUSSION TITLE
about how copy database file when db is opened

### DISCUSSION THREAD

**Discussion by xubobbs** [duckdb/duckdb#14806] [Q&A]
my server is running with duckdb file . I want to copy the db file to another dir and use my server run on this copyed db .
when I run ,it got the following error:

```
could not open database: duckdb error: INTERNAL Error: Failed to load metadata pointer (id 3218, idx 0, ptr 3218)

This error signals an assertion failure within DuckDB. This usually occurs due to unexpected conditions or errors in the program's logic.
```

what does this mean ? and How can I copy the db file when it is running ?

**Comment by Tishj** [comment]
You'd need to perform a checkpoint before you copy the db file, to ensure consistency of the system

**Reply by xubobbs** [reply]
how to do this when my server is running
